### ISSUE TITLE
Change visibility of $debug_info property

### ISSUE THREAD

**Issue by Revine-dev** [stefangabos/Zebra_Database#87]
Hello,

Can you please change visibility of the property `private $debug_info = array();` to protected please for heritage purpose please ?

Thank you
Rémi

**Comment by stefangabos** [comment]
Yes, I can - and I am working on it right now (I need to update my documentation generator script to highlight visibility)
Can you tell me why do you need this though?

**Comment by stefangabos** [comment]
Done, see 15c6bac.
I am still curios as to how do you plan on using this.
Thanks!

**Comment by Revine-dev** [comment]
My debug output is on Json format, so this property does not work how I need

**Comment by stefangabos** [comment]
yeah, i started working on it so that it doesn't tamper the ajax request's output

**Comment by Revine-dev** [comment]
If you can please add this property and the method to protected, it'd be great

**Comment by stefangabos** [comment]
I just pushed a proper working way (i think!) for debugging AJAX requests. Read [the docs](https://stefangabos.github.io/Zebra_Database/Zebra_Database/Zebra_Database.html#var$debug_ajax) about how to set it up. Let me know what you think.
